src/v1/components/inputs/Button/SecondaryButton.tsx

```diff
@@ -1,12 +1,21 @@
 import React from "react";
-import { Button, ButtonProps } from "@mui/material";
+import MUIButton, { ButtonProps as MUIButtonProps } from "@mui/material/Button";
 
-type SecondaryButtonProps = Omit<ButtonProps, "variant" | "color">;
+type SecondaryButtonProps = Omit<MUIButtonProps, "variant" | "color">;
 
-const SecondaryButton: React.FC<SecondaryButtonProps> = (buttonProps) => (
-  <Button variant="outlined" color="primary" {...buttonProps}>
-    {buttonProps.children}
-  </Button>
-);
+const SecondaryButton: React.FC<SecondaryButtonProps> = (buttonProps) => {
+  if (buttonProps.disableElevation) {
```

Comment:
Prompted thoughts for [future discussion](https://telicent.atlassian.net/wiki/spaces/FTS/pages/14352389/Bike+shed#Code-interpretations-of-Design-system-rules)

Reply:
I did speak to Justin about this but yes, we probably need alignment as some point when we have time. He is aware of this though